core/src/main/scala/caliban/validation/Validator.scala

```diff
@@ -34,10 +34,10 @@ object Validator {
   /**
    * A QueryValidation is a pure program that can access a Context, fail with a ValidationError or succeed with Unit.
    */
-  type QueryValidation            = EReader[Context, ValidationError, Unit]
+  type QueryValidation            = Context => EReader[Any, ValidationError, Unit]
```

Comment:
Technically a _very_ user-facing breaking change, but I'll be extremely surprised if we have any users that define custom validations and provide them to the config. In the very unlikely case that someone does this, it should be very straightforward to convert the validation to a function